packages/coinstac-ui/app/main/utils/boot/configure-browser-window.js

```diff
@@ -30,13 +32,32 @@ function createWindow() {
    * reference of the window object, if you don't, the window will be closed
    * automatically when the javascript object is GCed.
    */
+
+  splashWindow = new BrowserWindow({
+    width: 810,
+    height: 610,
+    frame: false,
+    alwaysOnTop: true
+  });
+
+  splashWindow.loadURL(`file://${renderSplashPath}`);
+
   mainWindow = new BrowserWindow({
     width: size.width,
     height: size.height,
+    show: false,
   });
 
   mainWindow.loadURL(`file://${renderIndexPath}`);
 
+  mainWindow.once('ready-to-show', () => {
+    mainWindow.show();
+    setTimeout(() => {
```

Comment:
this works for now, but we should hide on an IPC from main once the libs are loaded